src/commands/game_options/add.ts

```diff
@@ -281,7 +291,7 @@ export default class AddCommand implements BaseCommand {
                                     }
                                 ),
                         },
-                        interaction
+                        matchedGroups.length === 0 ? interaction : undefined
```

Comment:
Why no interaction response for `matchedGroups.length === 0`?

Reply:
The interaction is only passed when no groups are matched so this error message acknowledges the interaction. When at least one group is matched, then the options message acknowledges it. In short, this prevents duplicate ACKs